src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.Common/Metric.cs

```diff
@@ -12,10 +12,10 @@ namespace Microsoft.Extensions.Diagnostics.HealthChecks;
 
 internal static partial class Metric
 {
-    [Counter("healthy", "status", Name = @"R9\\HealthCheck\\Report")]
```

Comment:
I know that @xakep139 is also making changes around this on PR https://github.com/dotnet/extensions/pull/4482, so let's just make sure that these changes won't conflict with his.

Reply:
I see, reverting this one then